analyses/eth/docs/eth-risk-anlaysis.Rmd

```diff
@@ -0,0 +1,188 @@
+---
+title: '<img src="https://drive.google.com/uc?id=118y5T73-iSUZoAMtoJGddxq9QzD_GDKX" style="height:40px;float:left; margin:10px" /><img src="https://drive.google.com/uc?id=1fHQUzF3ZjaoHj9KQ33-94dK_X1hcmjzW" style="height:50px;float:left; margin:10px" />'
+pagetitle: 'ETH Baseline Risk Analysis' # Change title to something appropriate for your work
+output:
+  html_document:
+    css: ../../country_template/docs/style.css
+    includes:
+      in_header: ../../country_template/docs/header.html
+    df_print: paged
+    toc: yes
+    toc_float: yes
+    toc_depth: 4
+  pdf_document:
+    toc: yes
+    toc_depth: '4'
+editor_options:
+  chunk_output_type: inline
+---
+
+<br> 
+<br>
+<br>
+
+# Ethiopia Baseline Risk Analysis
+Analysis last updated 12-10-2021.
+
+<br>
+
+```{r setup, include = FALSE, message = FALSE, warning = FALSE}
+# source("") # Link to external script
+library(kableExtra)
+library(knitr)
+source('make_plots.R')
+knitr::opts_chunk$set(echo = FALSE) # do not print code by default
+knitr::opts_chunk$set(include = FALSE) # do not print output by default
+```
+
+
+This document is an initial exploration into risk of humanitarian need in Ethiopia. We're focusing most closely on climate-related risks, such as floods and drought. This document presents the outputs of basic data collection and analysis. It is intended to provide a starting point for a more in depth risk analysis for Ethiopia's upcoming Humanitarian Needs Overview. 
+
+## Historical Analysis
+
+### Frequency of past drought and flood events
+
+The figure below shows the national count of shock events per year, as recorded in EM-DAT. 
+
+```{r ts-basic, dpi=300, include=T, message= FALSE, warning=FALSE, fig.height=2, fig.align="center", fig.cap='National count of shock events per year'}
```

Comment:
You probably could put `message = FALSE` and `warning = FALSE` in your `knitr::opts_chunk$set()` in setup, as you probably never want those options there. Also, you put `include = FALSE` in your setup and then `include=T` in every individual block.